.github/workflows/macos-build.yml

```diff
@@ -29,11 +25,11 @@ jobs:
           node-version: 22
           cache: 'yarn'
       - name: Install node dependencies
-        working-directory: ${{ matrix.working-directory }}
+        working-directory: ${{ env.WORKING_DIRECTORY }}
         run: yarn
       - name: Install pods
-        working-directory: ${{ matrix.working-directory }}/macos
-        run: pod install
+        working-directory: ${{ env.WORKING_DIRECTORY }}/macos
+        run: bundle install && bundle exec pod install
       - name: Build app
-        working-directory: ${{ matrix.working-directory }}
-        run: yarn macos
+        working-directory: ${{ env.WORKING_DIRECTORY }}
+        run: npx react-native-macos run-macos
```

Comment:
Is this safe? I believe `npx` will run the latest available (cached?) version.

Reply:
Well, this is recommended in `react-native-macos` [docs](https://microsoft.github.io/react-native-windows/docs/rnm-getting-started#running-a-react-native-macos-app) 😅